This dates when libpthread was moved to glibc and some features were added.

--- a/libpthread/Versions	2013-09-21 22:32:47.069176263 +0200
+++ b/libpthread/Versions	2014-01-01 19:08:01.701025627 +0100
@@ -1,5 +1,5 @@
 libc {
-  GLIBC_2.13 {
+  GLIBC_2.13_DEBIAN_31 {
     pthread_attr_destroy; pthread_attr_getdetachstate;
     pthread_attr_getinheritsched; pthread_attr_getschedparam;
     pthread_attr_getschedpolicy; pthread_attr_getscope; pthread_attr_init;
@@ -131,8 +131,10 @@
     __pthread_spin_lock; __pthread_spin_trylock; __pthread_spin_unlock;
     _pthread_spin_lock;
   }
-  GLIBC_2.17 {
-    pthread_hurd_cond_wait_np;
-    pthread_hurd_cond_timedwait_np;
-  }
+  GLIBC_2.13_DEBIAN_38 {
+    pthread_hurd_cond_wait_np;
+  }
+  GLIBC_2.13_DEBIAN_39 {
+    pthread_hurd_cond_timedwait_np;
+  }
 }
--- a/Versions.def
+++ b/Versions.def
@@ -30,6 +30,8 @@
   GLIBC_2.11
   GLIBC_2.12
   GLIBC_2.13
+  GLIBC_2.13_DEBIAN_38
+  GLIBC_2.13_DEBIAN_39
   GLIBC_2.14
   GLIBC_2.15
   GLIBC_2.16
@@ -39,6 +39,7 @@
 %endif
   GLIBC_PRIVATE
   GLIBC_2.13_DEBIAN_19
+  GLIBC_2.13_DEBIAN_31
 }
 libcrypt {
   GLIBC_2.0
@@ -105,6 +106,8 @@
   GLIBC_2.6
   GLIBC_2.11
   GLIBC_2.12
+  GLIBC_2.13_DEBIAN_38
+  GLIBC_2.13_DEBIAN_39
   GLIBC_2.18
   GLIBC_PRIVATE
 }
